services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/commons/PpaDataRequestConverter.java

```diff
@@ -72,14 +72,11 @@ protected ExposureWindowsAtTestRegistration convertToExposureWindowAtTestRegistr
     PPAExposureWindow exposureWindow = newExposureWindow.getExposureWindow();
     Set<ScanInstancesAtTestRegistration> scanInstancesAtTestRegistration =
         convertToScanInstancesAtTestRegistrationEntities(newExposureWindow, technicalMetadata);
-    if (exposureWindow != null) {
```

Comment:
if exposure window would we null here , we will get a NPE int the ExposureWindowAtTestRegistration, exposureWindow.getDate() , right ?